libraries/helpers.rb

```diff
@@ -55,5 +55,40 @@ def management_type_lookup(profle_type)
       }
       mgmttype[profle_type]
     end
+
+# Generic convert to wsadmin array
```

Comment:
Could you add a comments with an example input/output?

Reply:
Done a simple example; boy are you in for a treat when you see what's coming in with our repository calling this.... ;)